lib/logger.js

```diff
@@ -1,7 +1,57 @@
 'use babel'
 
+import _ from 'lodash'
+
 export default class Logger {
-  error (/* statusCode, result, builder */) {}
-  warning (/* message */) {}
-  info (/* message */) {}
+  constructor () {
+    this.messages = []
+    this._group = null
+    const callback = () => { this.showFilteredMessages() }
+    atom.config.onDidChange('latex.loggingLevel', callback)
```

Comment:
This `Disposable` should probably to be captured, and disposed of in a `destroy` method or some such. Right now this base class (and hence anything that extends it) is leaky.

Thinking about it, do we actually need to listen for this? We should only care about this when we parse the log file. If someone changes the logging level, that should only affect subsequent builds, not historical ones.